config/manager/manager.yaml

```diff
@@ -13,20 +13,21 @@ metadata:
   name: controller-manager
   namespace: default
   labels:
-    control-plane: controller-manager
     app.kubernetes.io/name: eviction-autoscaler
+    app.kubernetes.io/component: controller
     app.kubernetes.io/managed-by: kustomize
 spec:
   selector:
     matchLabels:
-      control-plane: controller-manager
```

Comment:
you don't have to remove this one yet. I wonder if chanign any of these names breaks e2e. I forget if I reset up e2e as a block when I moved to azure.